source/code-generation/query-converter/query-management/download-queries.txt

```diff
@@ -0,0 +1,67 @@
+.. _rm-download-queries:
+
+================
+Download Queries
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Allows you to download completed conversions in the 
+:guilabel:`Query Converter` pane.
```

Comment:
(completenes)
Why do want to download queries? Can you add an agile style "so that you can ..." phrase to the end of this sentence?